tests/test_registry_management.py

```diff
@@ -202,26 +203,29 @@ def test_invalid_dimension_mapping(make_test_project_dir):
         orig_text = record_file.read_text()
 
         # Invalid 'from' record
-        record_file.write_text(orig_text + "invalid county,CO\n")
+        record_file.write_text(orig_text + "\ninvalid county,1,CO\n")
```

Comment:
This kind of problem happens when a file doesn't end with a newline character, which is a convention. I thought I had fixed this in the data. Note that git and GitHub complain about this.

Reply:
I might have ruined your fix in the test data.